lib/generators/shopify_app/app_proxy_controller/templates/app_proxy_controller.rb

```diff
@@ -0,0 +1,12 @@
+class AppProxyController < ApplicationController
+  if Rails.env.development? || Rails.env.test?
+   around_action :shopify_session
+  else
+   include ShopifyApp::AppProxyVerification
+  end
```

Comment:
Oh i see... you don't have access to Shopify during development? I always just use a dev shop and access the app proxy route through the shop.